src/components/todoList/todoList.css

```diff
@@ -0,0 +1,129 @@
+.todo-list-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 30px;
+    width: 100%;
+}
+
+.todo{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    width: 60%;
+    gap: 8px;
+}
+
+.todo-list{
+    display: flex;
+    align-items: center;
+    border: 2px solid transparent;
+    padding: 12px 8px;
+    border-radius: 8px;
+    background-color: #F2E471;
+    
+}
+.todo-list input[type="checkbox"]{
+    margin-right: 10px;
+    
+}
+
+.deletBtn{
+    margin-left: auto;
+}
+
+.deletBtn button{
+    border: none;
+    background-color: transparent;
+    font-size: 1.5rem;
+    transition: transform 0.3s ease;
+}
+
+.deletBtn button:hover {
+ cursor: pointer;
+        transform: rotate(-10deg);
+    
+}
+.completeAmount{
+    padding-bottom: 30px;
+}
+.completeAmount p, 
```

Comment:
You could stay consistent with naming your classes. You're using both camel and kebab-case in this project.